### ISSUE TITLE
`doctype` is uppercased after the plugin has run

### ISSUE THREAD

**Issue by XhmikosR** [sparanoid/grunt-sri-hash#4]
I use a lowercase `doctype` on purpose, but after running grunt-sri-hash `doctype` becomes `DOCTYPE`.

**Comment by sparanoid** [comment]
I use `jsdom` to serialize the whole DOM so I think it's `jsdom` does the transformation.

**Comment by XhmikosR** [comment]
Yes, correct :) But there must be some workaround. Maybe not using `serialize()` https://github.com/jsdom/jsdom#serializing-the-document-with-serialize?

**Comment by sparanoid** [comment]
Back from my vocation, sorry for my late reply.

`serialize()` seems to be the simplest way to get the working output. I have another grunt plugin that implements [a serialize option](https://github.com/sparanoid/grunt-assets-inline/blob/master/tasks/assets_inline.js#L389-L393). But the main purpose is to make fragment HTML output work.

I can port it here but it won't help get the `doctype` in proper case. You will just get the output without `doctype`. Do you have any better solution?

**Comment by XhmikosR** [comment]
Unfortunately, nope :/ It's not a big deal, if we want to be serious, but on the other hand, it's a choice the developer makes and this reverts it blindly.

Perhaps you could report this upstream? Maybe we get lucky and this gets fixed/worked around.

**Comment by sparanoid** [comment]
A workaround I can think of is to get and save current doctype and re-apply it after processing, but I'd like to keep simple. So here is https://github.com/jsdom/jsdom/issues/2386

**Comment by XhmikosR** [comment]
Thanks! I'm gonna follow that issue.

**Comment by XhmikosR** [comment]
I guess the specs can't change at this point so it either needs to be handled here (ideally) or I could just hook up another plugin to do the replacement.

**Comment by sparanoid** [comment]
@XhmikosR Just wrote a separate plugin to transform DOCTYPE, check it out: https://github.com/sparanoid/grunt-doctype

**Comment by XhmikosR** [comment]
Thanks, I'm gonna try it soon!
